integration/kotlinx-coroutines-guava/src/module-info.java

```diff
@@ -0,0 +1,8 @@
+module kotlinx.coroutines.guava {
+    requires kotlin.stdlib;
+    requires kotlinx.coroutines.core;
+    requires com.google.common;
+    requires failureaccess;
```

Comment:
What is `failureaccess` and why it is needed?

Reply:
This is a transitive dependency of Guava. One of the classes inside it is used in the `kotlinx-coroutines-guava` module. 
More specifically, there is a typecheck for `com.google.common.util.concurrent.internal.InternalFutureFailureAccess`.

Reply:
Compilation works fine without this `requires failureaccess`, so I removed it.